ibis/backends/flink/registry.py

```diff
@@ -282,6 +282,21 @@ def _floor_divide(translator: ExprTranslator, op: ops.Node) -> str:
     return f"FLOOR(({left}) / ({right}))"
 
 
+def _array(translator: ExprTranslator, op: ops.arrays.Array) -> str:
+    return "ARRAY[{}]".format(", ".join(map(translator.translate, op.exprs)))
+
+
+def _array_contains(translator: ExprTranslator, op: ops.arrays.ArrayContains) -> str:
```

Comment:
This can be done using `fixed_arity("ARRAY_CONTAINS", 2)` I think.

Reply:
Thanks, replaced with `fixed_arity`.